### DISCUSSION TITLE
My github account was blocked, without any reasons

### DISCUSSION THREAD

**Discussion by freekos** [vercel/community#1302] [Help] [answered]
### Summary

Please unblock my github account in vercel

### Additional information

When i gonna to sign up or sign in in vercel, with my github account, i have got a next message: This user account is blocked. Contact registration@vercel.com for more information. I don't know why? Because i didn't use vercel before it

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble signing in. 🙁 If you send a message to [registration@vercel.com](mailto:registration@vercel.com) with information about the error message you're seeing, your GitHub username, and what you're trying to deploy, the team can help you sort out the problem

**Reply by freekos** [reply]
ok thx)
